<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>CALENDAR</string>
    </property>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>312</width>
       <height>183</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_SelectedDate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Selected Date:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_Date">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ShowEvents">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Events</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>641</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>SELECT AN OPTION TO ADD DATA</string>
    </property>
    <widget class="QPushButton" name="pushButton_AddEvent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Event</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_AddSubjects">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Subjects</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_References">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add References</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Syllabus">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Syllabus</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_GPAcalculator">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>GPA calculator</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Schedule">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Schedule</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView_Event">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>641</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>641</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>SELECT AN OPTION TO RETRIEVE DATA</string>
    </property>
    <widget class="QPushButton" name="pushButton_ViewSubjects">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>View Subjects</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>121</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>RETRIEVE SYLLABUS</string>
     </property>
     <widget class="QComboBox" name="comboBox_Syllabus">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_RetrieveSyllabus">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Retrieve</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_Load">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>120</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>RETIEVE REFERENCES</string>
     </property>
     <widget class="QComboBox" name="comboBox_References">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_References2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_RetrieveReferences">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Retrieve</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>VIEW SCHEDULE</string>
     </property>
     <widget class="QComboBox" name="comboBox_Schedule">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Sunday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Monday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tuesday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wednesday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Thursday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Friday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Saturday</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_viewSchedule">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>View Class Schedule</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
